### PR TITLE
Fixed the old "hello word" business...

### PR DESCRIPTION
**Pull request by SystemInvecklare**
Regained honor. @gnatamania have a look snug-snug!

### CODE REVIEW

**Comment by SystemInvecklare** [comment]
@gnatamania You mean like a vacation?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # hello-word
-Hello word! I mean world! Oh dang...
+Hello world!
```
Comment (gnatamania) [inline README.md]: Capital W!!!!

Comment (SystemInvecklare) [inline README.md]: Will get right on it!

**Comment by SystemInvecklare** [comment]
How 'bout now?

**Comment by gnatamania** [comment]
LGTM!!

**Comment by SystemInvecklare** [comment]
Thankkkkkssssss!

**Pull request merged**